### DISCUSSION TITLE
v0.19.0 release

### DISCUSSION THREAD

**Discussion by Conengmo** [python-visualization/folium#2040] [Collaborators]
We've got some nice PR's in, let's do a release! Here's the draft: https://github.com/python-visualization/folium/releases/tag/untagged-37303afa0efd9a1098fe

@ocefpaf @hansthen any comments or good to go?

**Comment by Conengmo** [comment]
And a corresponding release in Branca: https://github.com/python-visualization/branca/releases/tag/untagged-338db33b05341e04c060

Since it just contains that Mypy support file I opted to go for v0.8.1.

**Comment by ocefpaf** [comment]
Both LGTM! Thanks!

**Comment by Conengmo** [comment]
Let's go then! 🚀  Thanks for the check!
